content/pl/docs/tasks/_index.md

```diff
@@ -5,80 +5,13 @@ weight: 50
 content_type: concept
 ---
 
-{{< toc >}}
-
 <!-- overview -->
 
 W tej części dokumentacji Kubernetesa znajdują się opisy
 sposobu realizacji różnych zadań. Przedstawione są one zazwyczaj jako
 krótka sekwencja kilku kroków związanych z pojedynczym zadaniem.
 
-
```

Comment:
To wyparowało czy przeniesione gdzies indziej?

Reply:
w oryginale została sama linkownia